library/StandardLibrary/src/Juvix/Library/Feedback.hs

```diff
@@ -0,0 +1,61 @@
+module Juvix.Library.Feedback where
+
+import qualified Control.Monad.Trans as Trans
+import Juvix.Library
+
+-- | Keep track of messages during the compilation or REPL.
+-- The implementation is based on
+-- [[https://github.com/UU-ComputerScience/uu-cco/blob/master/uu-cco/src/CCO/Feedback.hs]].
+data Feedback msg a
+  = Success msg a -- Indicate success.
```

Comment:
I can't see any use of the `msg` parameter in Success. In which cases are we planning to provide a message to Success?

Reply:
In case of success, there are still reasons to provide feedback. Compilers typically present warnings or advice, even though the compilation is successful.

Reply:
Yes, although using it in its current form looks difficult. Have you considered having a Writer instance?